interop/client/client.go

```diff
@@ -84,19 +86,26 @@ const (
 	credsTLS
 	credsALTS
 	credsGoogleDefaultCreds
+	credsComputeEngineChannelCreds
 )
 
 func main() {
 	flag.Parse()
-	var useGDC bool // use google default creds
+	var useGDC bool                       // use google default creds
```

Comment:
For consistency, please either spell GDC out or abbreviate ComputeEngineChannelCreds in the next line :)

Reply:
Done. Abbreviated `useComputeEngineChannelCreds` to `useCEC`